### DISCUSSION TITLE
Start zigbee2mqtt without sniffer?

### DISCUSSION THREAD

**Discussion by rubentest** [Koenkk/zigbee2mqtt#10213] [General]
I'm trying to start zigbee2mqtt in a new machine without using at this momnet the zigbee CC controller.
It's possible to start it? 
I tried to remove serial: in config but isn't working.
Any other possiblity ?
Many thanks

**Comment by Koenkk** [comment]
This is not possible

**Reply by bartplessers** [reply]
Hi @Koenkk ,
I remember asking the same question a while ago, and there was some enthousiasm for this feature by other uses as well...

Will you ever consider implementing this?

Grtz
B

**Reply by Koenkk** [reply]
Maybe in the future, but currently low prio for me.
